src/services/mentees.js

```diff
@@ -58,10 +58,33 @@ module.exports = class MenteesHelper {
 
 		const totalSession = await sessionAttendeesQueries.countEnrolledSessions(id)
 
+		const [firstRole] = roleId
+		const userRoles = firstRole?.id
```

Comment:
User can have multiple roles, this doesnt work with that, use map to extract all the ids